src/components/OnboardComponents/views.js

```diff
@@ -8,15 +8,18 @@ export const renderSpinner = () => (
   </div>
 );
 
-export const pushToDashboard = history => history.push("/dashboard");
+export const pushToDashboard = (history) => history.push("/dashboard");
 
-export const renderManualBtn = ({ handleClick, error }) => (
-  <div className="main">
-    <p className="error">Sorry Please Try Again</p>
-    <div className="manualBudgetButton">
-      {error ? error : " "}
```

Comment:
This line was causing a rendering error to be thrown from React, it was attempting to render an object